packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayoutNew/CoinmarketLayoutNavigation/CoinmarketLayoutNavigation.tsx

```diff
@@ -34,6 +48,24 @@ const CoinmarketLayoutNavigation = () => {
                 />
             ) : null}
 
+            {isInEEA ? (
+                <>
+                    <SeparatorWrapper>
+                        <Divider
+                            orientation="vertical"
+                            strokeWidth={1}
+                            color="borderElevation0"
```

Comment:
Do you have to set this color here? Shouldn't it work without setting it? It should get elevated color

Reply:
yeah, without it, it's a different color `#EEEEEE` instead of `#E2E2E2`

Reply:
this would result in a different color than we want

Reply:
ok, removed